index.js

```diff
@@ -51,6 +48,16 @@ module.exports = async app => {
   async function forRepository (context) {
     let config = await getConfig(context, 'stale.yml')
 
+    let interval = 1
+    if (config && config.interval) {
+      interval = config.interval
+    }
+
+    // Visit all repositories to mark and sweep stale issues
+    let scheduler = createScheduler(app, {
+      interval: interval * 60 * 60 * 1000
+    })
```

Comment:
Schedulers currently scan _all_ repositories for _all_ installations. So this would create a duplicate scheduler for every single repository, but that scheduler is not isolated to the single repository.